ccm

```diff
@@ -8,6 +8,9 @@ from ccmlib import common
 from ccmlib.cmds import command, cluster_cmds, node_cmds
 from ccmlib.remote import PARAMIKO_IS_AVAILABLE, get_remote_usage, get_remote_options, execute_ccm_remotely
 
+import os
+import warnings
```

Comment:
This is the wrong place for these imports. You could install and run `isort ccm` against this file to have it automatically cleaned, but stdlib imports belong first. So:
```
import os
import sys
import warnings

import pkg_resources
...
```